kobo/apps/trash_bin/utils.py

```diff
@@ -298,6 +290,35 @@ def replace_user_with_placeholder(
     return placeholder_user
 
 
+@contextmanager
+def signals_temporarily_disconnected(save=False, delete=False):
```

Comment:
Nitpick, but let's make this `temporarily_disconnect_signals()`, in the imperative mood